problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -175,8 +175,15 @@ function MentoringWithStepsBlock(runtime, element) {
         reviewTipsDOM.empty().hide();
     }
 
+    function updateNext() {
```

Comment:
@Kelketek Nit: I think it would be good to rename this to something a little less generic, like `updateNextLabel` or even `updateNextButtonLabel`.